app/server.rb

```diff
@@ -0,0 +1,27 @@
+class Chitter < Sinatra::Base
+
+  enable :sessions
+  register Sinatra::Flash
+  register Sinatra::Partial
+  use Rack::MethodOverride
+  set :session_secret, 'rochefort rocks'
+  set :partial_template_engine, :erb
+  set :public_folder, Proc.new { File.join(root, 'static') }
```

Comment:
Style/Proc: Use `proc` instead of `Proc.new`. (https://github.com/bbatsov/ruby-style-guide#proc)